services/order.py

```diff
@@ -0,0 +1,33 @@
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, User
+from services.movie_session import get_movie_session_by_id
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None,
+) -> None:
+    with transaction.atomic():
```

Comment:
Use `transaction.atomic` as decorator instead of context manager. It makes your code more readable

Reply:
I don't really understand you